addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,11 +1,47 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, api
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
+    slide_ids = fields.One2many('slide.slide', 'survey_id', 'Certification')
+    slide_channel_ids = fields.One2many('slide.channel', compute='_compute_slide_channel_ids')
+    courses_count = fields.Integer("Linked Courses", compute="_compute_slide_channel_ids")
```

Comment:
I would add `groups='survey.group_survey_user'` to both fields to hide them for non-survey users.

Reply:
Should be website_slides.group_website_slides_officer instead

Reply:
Done